docs/operational_guide/multiple_m3db_clusters.md

```diff
@@ -0,0 +1,125 @@
+## Write to multiple M3DB clusters via M3Coordinator
+
+### Overview:
+
+Default M3 architecture has the M3Coordinator writing to and aggregating meterics from a single M3DB cluster. If wanting to add more than one, follow the below instructions. 
+
+Use case(s):
+- Sending metrics to different namespaces for different retention periods, etc.
```

Comment:
Hmm this kind of makes it seem that the only way to write metrics to multiple namespaces is through this setup. Maybe like a use case similar to this would be having a long term and short term m3db cluster? @robskillington thoughts?

Reply:
Maybe add a note that this is specifically for a single coordinator in the use case?